src/docs/tutorials/04-edit-email-templates-with-flex-cli/index.md

```diff
@@ -188,19 +188,88 @@ text editor and make a simple change:
 
 When you've made the change, save the file.
 
+## Preview your changes
+
+You can test your changes and templates by previewing or sending a test
+email using your local template files. To preview the change above:
+
+```bash
+flex-cli notifications preview -m my-test-marketplace --template preauth-nightly-booking/templates/new-booking-request
+```
+
+The command will render the given template and open a browser tab with
+the output HTML:
+
+![Email preview in a browser](./preview-browser.png)
+
+Note the subject in the tab title. It will also output the rendered text
+version to the terminal:
+
+![Email preview in terminal](./preview-terminal.png)
+
+You can now use your normal browser developer tooling to test changes to
+the template and verify how the responsiveness of the content works.
+
+If you want to verify the email in various email software, you can also
+send the test preview email:
+
+```bash
+flex-cli notifications send -m my-test-marketplace --template preauth-nightly-booking/templates/new-booking-request
+```
+
+The email is sent to the email address of the admin user that was used
+in logging in to the CLI:
+
+![Email preview in inbox](./preview-inbox.png)
+
+### Using custom context variables in templates
+
+As you see from the previews above, the templates are rendered with the
+fixed sample context. The `notifications preview` and
```

Comment:
Ok now reading further I see you mention the using of fixed sample context here. This works but do we trust users to keep reading this far before introducing the sample context?